src/components/Flex/Flex.stories.tsx

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { Flex, FlexProps } from './Flex';
+import { Box } from '@components/Box';
+
+const DirectionTypes = ['tw-flex-row', 'tw-flex-col', 'tw-flex-row-reverse', 'tw-flex-col-reverse'];
```

Comment:
the `*Types` values shouldn't have the tailwind classes, but the possible options defined on the component file, like `['row', 'column'...]`